dist/leaflet.draw-src.js

```diff
@@ -1311,10 +1479,10 @@ L.Edit.Poly = L.Handler.extend({
 
 	_createMiddleMarker: function (marker1, marker2) {
 		var latlng = this._getMiddleLatLng(marker1, marker2),
-		    marker = this._createMarker(latlng),
-		    onClick,
-		    onDragStart,
-		    onDragEnd;
+			marker = this._createMarker(latlng),
+			onClick,
+			onDragStart,
+			onDragEnd;
```

Comment:
These actually translate to tabs - not sure what was in before but the entire project is tab rather than space-tabs.